doc/about.rst

```diff
@@ -22,11 +22,22 @@ Governance
 The decision making process and governance structure of scikit-learn is laid
 out in the :ref:`governance document <governance>`.
 
-Authors
--------
+.. _authors:
+
+The people behind scikit-learn
+-------------------------------
+
+Scikit-learn is a community project, developed by a large group of
+people, all across the world. A few teams, listed below, have central
+roles, however a more complete list of contributors can be found `on
+github
+<https://github.com/scikit-learn/scikit-learn/graphs/contributors>`__.
 
-The following people are currently core contributors to scikit-learn's development
-and maintenance:
+Maintainers Team
+................
+
+The following people are currently maintainers, in charge of
+consolidating scikit-learn's development and maintenance:
 
 .. include:: authors.rst
```

Comment:
Do we also want to rename `authors.rst`?

Reply:
Done!

I hope that I did not screw something up here. I guess that we'll know next time we update the contributor list, eg when updating the emeritus